docs/sources/shared/flow/reference/components/authorization-block.md

```diff
@@ -7,7 +7,7 @@ headless: true
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `type` | `string` | Authorization type, for example, "Bearer". | | no
-`credential` | `secret` | Secret value. | | no
+`credentials` | `secret` | Secret value. | | no
```

Comment:
this was documented incorrectly compared to the river type. Alternatively, I can update the river type to match the docs if we prefer `credential`

Reply:
No, I think `credentials` is probably what was intended.